infra/aws/terraform/audit-account/guarduty.tf

```
@@ -0,0 +1,53 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+resource "aws_guardduty_detector" "default" {
+  provider = aws.audit
+  enable   = true
+
```

```suggestion

  finding_publishing_frequency = "ONE_HOUR"

```
Whatever interval we choose, it's good to define this in code so that Terraform then reports on any drift.

---

Implementation will done in a separate PR.